data/data/aws/variables-aws.tf

```diff
@@ -69,3 +69,20 @@ variable "aws_worker_availability_zones" {
   description = "The availability zones to provision for workers.  Worker instances are created by the machine-API operator, but this variable controls their supporting infrastructure (subnets, routing, etc.)."
 }
 
+variable "aws_vpc" {
+  type = string
+  default = null
+  description = "(optional) An existing network (VPC ID) into which the cluster should be installed."
+}
```

Comment:
why so we need this?
https://www.terraform.io/docs/providers/aws/d/subnet.html#vpc_id

we should be able to get the id for VPC using subnet data source..

Reply:
> we should be able to get the id for VPC using subnet data source..

Yup, but we will have already extracted the VPC ID in Go when validating the subnets, so I thought we might as well just pass it along.